scripts/build-dist.js

```diff
@@ -0,0 +1,77 @@
+var _ = require('lodash');
+var harp = require('harp');
```

Comment:
This isn't used...

Why not `harp compile`?

http://harpjs.com/docs/environment/compile